changelog.md

```diff
@@ -1,3 +1,29 @@
+# 0.2.1 Release
+This is the second release for the Ternoa Testnet.
+The two major features are the two new pallets capsule and associated-accounts.
+
+# 🚀 New Features
+The two major features are the two new pallets **capsule** and **associated-accounts** respectivelify for
```

Comment:
Same as above, there are many more features that we did (like adding links to the marketplace website) and this needs to be here in a way that a grandma would understand it.

Reply:
Having new pallets isn't a feature descriptions.

Instead of `- usernames management` you could write "It is now possible to link third party accounts with your main account"

Reply:
✅